x-pack/plugins/security_solution/public/flyout/document_details/right/footer.tsx

```diff
@@ -12,10 +12,17 @@ import { FlyoutFooter } from '../../../timelines/components/side_panel/event_det
 import { useRightPanelContext } from './context';
 import { useHostIsolationTools } from '../../../timelines/components/side_panel/event_details/use_host_isolation_tools';
 
+interface PanelFooterProps {
+  /**
+   * Boolean that indicates whether footer is read only and action should be hidden
+   */
+  isReadOnly: boolean;
```

Comment:
any reason why you didn't use `isPreview` here instead of `isReadOnly`? I'm not decided on this yet, but I feel like we could stick to `isPreview` everywhere in our code, and only switch to other names right when we use external components. I feel like if you make tracing code easier?

So basically we would do `isReadOnly={isPreview}` below...

Let me know what you think!